client/webpack.config.js

```diff
@@ -0,0 +1,100 @@
+var path = require('path');
+var webpack = require('webpack');
+var configData = require('./.config.json')
+var glob = require('glob')
+var _ = require('lodash')
+var CopyWebpackPlugin = require('copy-webpack-plugin')
```

Comment:
Single `var` statement would work too, right?

End of lines are not consistent.